docroot/sites/classrooms.uiowa.edu/modules/classrooms_core/classrooms_core.module

```diff
@@ -31,6 +55,199 @@ function classrooms_core_form_alter(&$form, FormStateInterface $form_state, $for
+      $variables['title_attributes']['class'] = 'h4 headline headline--serif headline--underline';
+      break;
+
+    // Add fivecol grid class to image gallery.
+    case 'field_block:node:room:field_gallery_images':
+      if (!isset($variables['attributes']['class'])) {
+        $variables['attributes']['class'] = [''];
+      }
+      $variables['attributes']['class'][] .= 'grid--fivecol--20 photoswipe-gallery';
+      break;
+
+    // Add inline class for blocks to appear inline.
+    case 'field_block:node:room:field_room_type':
+    case 'field_block:node:room:field_room_seating_chart':
+    case 'field_block:node:room:field_room_max_occupancy':
+      if (!isset($variables['attributes']['class'])) {
+        $variables['attributes']['class'] = [''];
+      }
+      $variables['attributes']['class'][] .= 'element--inline block-margin__top';
+      break;
+
+    // Add inline class for blocks to appear inline.
+    case 'field_block:node:room:field_room_map':
+      if (!isset($variables['attributes']['class'])) {
+        $variables['attributes']['class'] = [''];
+      }
+      $variables['attributes']['class'][] .= 'element--cover';
+      break;
+
+    // Center video.
+    case 'field_block:node:room:field_room_video':
+      if (!isset($variables['attributes']['class'])) {
+        $variables['attributes']['class'] = [''];
+      }
+      $variables['attributes']['class'][] .= 'align-center';
+      break;
+
+  }
+}
+
+/**
+ * Implements hook_preprocess_field().
+ */
+function classrooms_core_preprocess_field(&$variables, $hook) {
+  switch ($variables['element']['#field_name']) {
+    case 'field_room_technology_features':
+    case 'field_room_tile_details':
+    case 'field_room_design_details':
+    case 'field_room_classroom_furniture':
+    case 'field_room_accessibility':
+    case 'field_room_features':
+      if (!isset($variables['title_attributes']['class'])) {
+        $variables['title_attributes']['class'] = '';
+      }
+      $variables['title_attributes']['class'] .= 'h6 headline headline--serif headline--underline';
+      break;
+
+    case 'field_technology_features_icon':
+      if (!isset($variables['attributes']['class'])) {
+        $variables['attributes']['class'] = '';
+      }
+      $variables['attributes']['class'] .= 'fa-li';
+      break;
+  }
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function classrooms_core_preprocess_layout(&$variables) {
+  /** @var \Drupal\Core\Layout\LayoutDefinition $layout */
+  $layout = $variables['layout'];
+
+  // Process the header layout.
+  if ($layout->id() === 'layout_header') {
+    $node = \Drupal::routeMatch()->getParameter('node');
+    $node = ($node ?? \Drupal::routeMatch()->getParameter('node_preview'));
+    if ($node instanceof NodeInterface) {
+      if (in_array($node->getType(), [
+        'room',
+      ])) {
+        if (!isset($variables['banner_attributes'])) {
+          $variables['banner_settings'] = [];
+        }
```

Comment:
Is it intentional that this is checking the existence of a different array key than it then initializes?

Reply:
Good question.  I was reusing previous code for setting up our featured image.

Reply:
My guess is that this was a copy/paste error whenever it was originally written.

Reply:
Thanks for pointing this out!  I have updated the code.